converters/toZigbee.js

```diff
@@ -5368,6 +5368,23 @@ const converters = {
             }
         },
     },
+    tuya_do_not_disturb: {
+        key: ['do_not_disturb'],
+        convertSet: async (entity, key, value, meta) => {
+            await entity.command('lightingColorCtrl', 'tuyaDoNotDisturb', {enable: value ? 1 : 0});
+            return {state: {do_not_disturb: value}};
+        },
+    },
+    tuya_color_power_on_behavior: {
+        key: ['tuya_color_power_on_behavior'],
```

Comment:
Having `color_power_on_behavior` sounds better in my opinion.

Reply:
Could we have this under `color_options` ? https://github.com/Koenkk/zigbee-herdsman-converters/blob/1c60448990c69d0737268381fcac6e8ce5253748/converters/toZigbee.js#L100 we use that for some of the generic lightingColorCtrl options, so having it use the same key would be more consistant?

Reply:
@sjorge look at https://github.com/Koenkk/zigbee-herdsman/pull/390
it is not common case - it it tuya commands

Reply:
Both are good, since I don't expect any other options this is fine.